_startup/retraite.md

```diff
@@ -1,12 +1,16 @@
 ---
-title: Mes démarches retraite
+title: Mes démarches retraite pas à pas
 mission: Obtenez en quelques clics la liste personnalisée de vos démarches de retraite.
 domain: retraite.beta.gouv.fr
 repository: https://github.com/sgmap/retraite
 status: construction
 contact: contact@retraite.beta.gouv.fr
 start: 2015-10-01
-owner: CNAV
-screenshot: false
+owner: CNAV, MSA, RSI
 ---
 
+L'application "Mes démarches retraite pas à pas" détermine la liste personnalisée des démarches à suivre pour préparer au mieux son départ à la retraite. 
+
+Le futur retraité saisit quelques informations (nom de naissance, date de naissance, et numéro de sécurité sociale), puis répond à quelques questions sur sa carrière et sa situation actuelle. Il obtient alors sa liste personnalisée des démarches qu'il peut télécharger et imprimer. Cette liste pourra l'accompagner tout au long de la préparation de sa retraite.
```

Comment:
C'est toi qui vois, je te donne ma meilleure idée :)

Sent from my iPhone

> On 6 juil. 2016, at 18:08, Xavier Nopre <notifications@github.com> wrote:
> 
> In _startup/retraite.md:
> 
> >  ---
> >  
> > +L'application "Mes démarches retraite pas à pas" détermine la liste personnalisée des démarches à suivre pour préparer au mieux son départ à la retraite. 
> > +
> > +Le futur retraité saisit quelques informations (nom de naissance, date de naissance, et numéro de sécurité sociale), puis répond à quelques questions sur sa carrière et sa situation actuelle. Il obtient alors sa liste personnalisée des démarches qu'il peut télécharger et imprimer. Cette liste pourra l'accompagner tout au long de la préparation de sa retraite.
> @Morendil : Merci Laurent pour ces retours. Moi je veux bien faire comme ça, si c'est comme ça qu'il faut faire, et la mixité est bien légitime, néanmoins, lorsque je lis des textes avec ces formulations, je trouve ça lourd et très illisible... Qu'en penses-tu ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>